sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -410,6 +410,9 @@ def _add_argparse_args(cls, parser):
                         'Experimental. '
                         'See https://cloud.google.com/dataflow/pipelines/'
                         'updating-a-pipeline')
+    parser.add_argument('--enable_streaming_engine',
```

Comment:
Do you need action=store_true (e.g. https://github.com/apache/beam/blob/master/sdks/python/apache_beam/options/pipeline_options.py#L300) ?

Reply:
added